src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -33,4 +33,11 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
         return Optional.ofNullable(relevantPersons);
     }
 
+    /**
+     *  Returns the string the gives the feedback to the user
+     */
+    public String getFeedbackToUser () {
```

Comment:
Coding standard violation. There shouldn't be spaces between the brackets and method name.